pkg/endpoint/endpoint.go

```diff
@@ -82,6 +83,12 @@ var (
 		Requires:    []string{OptionConntrack},
 	}
 
+	OptionSpecConntrackLocal = option.Option{
+		Define:      "CONNTRACK_LOCAL",
+		Description: "Use container local connection tracking table instead of global one",
```

Comment:
While I was reading the code I was asking my self "What is local? Do we have a non-local CT?" Now I get it, can I purpose to change the name to `Use container dedicated connection tracking..`? It's just a thought